packages/admin/src/FilamentManager.php

```diff
@@ -181,6 +182,26 @@ public function getResources(): array
         return array_unique($this->resources);
     }
 
+    public function getResource(Model | string $model): ?Resource
+    {
+        $modelFqcn = ($model instanceof Model)
+            ? $model::class
+            : $model;
+
+        foreach ($this->getResources() as $resourceClass) {
+            /**
+             * @var Resource $instance
+             */
+            $instance = new $resourceClass();
+
+            if ($modelFqcn === $instance::getModel()) {
```

Comment:
The `new $resourceClass` is pointless because you're just calling a static method. You can do that on the `$resourceClass` already.